packages/sdks-tests/src/e2e-tests/custom-breakpoints.spec.ts

```diff
@@ -115,17 +115,24 @@ breakpoints: {
     });
 
     test('extra small mobile size', async ({ page, sdk }) => {
-      test.skip(excludeGen2(sdk));
       await page.setViewportSize({ width: 320, height: 1000 });
       await page.goto('/custom-breakpoints');
 
+      let expectedTextColor = 'rgb(25, 201, 216)'; // blueish text color
+      if (checkIsRN(sdk)) {
```

Comment:
can we skip the test entirely and add a comment that react-native doesn't support this?

Reply:
Yeah, skipping the test at the very start would make it more explicit that RN SDK does not support this feature yet.

Reply:
done